### DISCUSSION TITLE
How to make the background transparent

### DISCUSSION THREAD

**Discussion by Turhvjbufv** [anddea/revanced-patches#585] [Q&A] [answered]
![Screenshot_20240607-174047_RVX_1.png](https://github.com/anddea/revanced-patches/assets/91851160/463aca6a-7b72-441e-8e69-8ee1263c7496)
Setting custom player overlay opacity to 0 does nothing(it does other stuff not this)
Latest current release

**Comment by anddea** [comment, marked as answer]
You have to include `Hide player buttons background` patch, it will force remove the background.
